src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -294,10 +306,51 @@ protected void setupCustomRouting(@NonNull Config cfg) {
         patch(UI_ROOT + RoutePath.USER_SETTINGS_REQUEST, new UserSettingRoute(), new JsonTransformer());
 
         setupJobs(cfg, kitUtil, notificationUtil, eventUtil, container, receiver);
+        
+        setupPubSub();
 
         logger.info("Finished setting up DSM custom routes and jobs...");
     }
 
+    private void setupPubSub() {
+        String projectId = "hl7-dev-274911";
+        String subscriptionId = "pegah-results-sub";
```

Comment:
you could use the static strings =)
maybe we could also move them to the config file